app/src/main/res/values/colors.xml

```diff
@@ -5,6 +5,160 @@
     <color name="purple_700">#FF3700B3</color>
     <color name="teal_200">#FF03DAC5</color>
     <color name="teal_700">#FF018786</color>
-    <color name="black">#FF000000</color>
-    <color name="white">#FFFFFFFF</color>
+    <color name="white">#FFFFFF</color>
```

Comment:
try to put only the colors you are going to use, it's fine if you can keep some extra ones, but try to stay as simple as possible

Reply:
Extra colors have been removed.